src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -1007,6 +1013,8 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
         const val SNYKCODE_SECURITY_ISSUES_ROOT_TEXT = " Code Security"
         const val SNYKCODE_QUALITY_ISSUES_ROOT_TEXT = " Code Quality"
         const val IAC_ROOT_TEXT = " Configuration Issues"
+        const val CONTAINER_ROOT_TEXT = " Container Vulnerabilities"
```

Comment:
As said, I think it might be better to name it "Container Issues"

Reply:
Answered in general comment below